llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -207,6 +207,11 @@ cl::opt<bool> DeviceGlobals{
     cl::desc("Lower and generate information about device global variables"),
     cl::cat(PostLinkCat)};
 
+cl::opt<bool> HostPipes{
```

Comment:
Do we really need this flag? It seems like it is not used in any way

Reply:
Yep, sorry, this is leftover from when CompileTimeProperties was still part of sycl-post-link. I'll remove.